package.json

```diff
@@ -102,13 +102,16 @@
     "jsdom": "^13.2.0",
     "less": "^3.0.4",
     "less-loader": "^4.1.0",
+    "luxon": "^1.21.3",
     "style-loader": "^0.21.0",
     "timeline-plus": "^2.3.6",
     "typeface-varela-round": "0.0.54",
     "vue-awesome": "^3.5.1",
     "vue-chartjs": "^3.4.2",
     "vue-d3-sunburst": "^1.0.0",
+    "vue-datetime": "^1.0.0-beta.11",
     "vue-property-decorator": "^8.3.0",
-    "vuex": "^3.1.1"
+    "vuex": "^3.1.1",
+    "weekstart": "^1.0.1"
```

Comment:
Can't see where this dependency is used?

Reply:
It's a peer dependency of the datetime picker.